appveyor.yml

```diff
@@ -18,9 +30,22 @@ before_build:
 
     gulp coalesce
 
-    cd ..\Coalesce.Domain
-
-    dotnet ef database update
+    cd ..\..
 build:
+  parallel: true
   project: Coalesce.sln
-  verbosity: minimal
\ No newline at end of file
+  verbosity: minimal
+after_build:
+  - dotnet pack "src\IntelliTect.Coalesce\IntelliTect.Coalesce.csproj" -c %CONFIGURATION% --no-build --include-symbols --output "%APPVEYOR_BUILD_FOLDER%\artifacts"
+  - dotnet pack "src\IntelliTect.Coalesce.CodeGeneration\IntelliTect.Coalesce.CodeGeneration.csproj" -c %CONFIGURATION% --no-build --include-symbols --output "%APPVEYOR_BUILD_FOLDER%\artifacts"
+  - dotnet pack "src\IntelliTect.Coalesce.Cli\IntelliTect.Coalesce.Cli.csproj" -c %CONFIGURATION% --no-build --include-symbols --output "%APPVEYOR_BUILD_FOLDER%\artifacts"
+artifacts:
+- path: artifacts\**\*.*
+deploy:
+  provider: NuGet
+  server: https://ci.appveyor.com/nuget/intellitect-jamnmeof65y0/api/v2/package 
+  symbol_server: https://ci.appveyor.com/nuget/intellitect-jamnmeof65y0/api/v2/package
+  api_key:
+    secure: 6BCd6W92HLY2RV4bTQSKNue9eoB7ETEYbvihtXy8T5k=
```

Comment:
That's an encrypted version of it.  AppVeyor provides neat UI for it: https://ci.appveyor.com/tools/encrypt